README.md

```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
 
+## Resources
+Trello Board [https://trello.com/b/M9VUQl9S/nodejs-collection](https://trello.com/b/M9VUQl9S/nodejs-collection)
+
+### Key Roles:
+- **Contributor/Writer** : Person submitting article or idea for inclusion on the Node.js Collection
+- **Editor** : Technical Reviewer. Person who ensures the content's technical integrity.
+
 ## Guidelines/General Tips of Submissions
 
-- Posts from members and community members must be vendor neutral, though they may mention vendors involved in specific project deployments if this serves the purpose of educating readers. 
-- Posts about hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, are allowed, but again shouldn’t feel like an advertisement for your services or company. 
+- Posts from members and community members must be vendor-neutral, though they may mention vendors involved in specific project deployments if this serves the purpose of educating readers.
+- Posts about hosting of an in-person event or speaking at an event, or other indications of meaningful participation in the community, are allowed, but again shouldn’t feel like an advertisement for your services or company.
 - Your post must be your original content, but can be published on the Node.js Foundation Medium publication, if you received permission from the editors of the website to do so.
 - All content should have a byline (preferably by a company engineer) and be published with Creative Commons with Attribution, so you’re welcome to republish on your blog.
-- The most interesting posts are ones that teach or explain how to do something in a way maybe others haven’t thought of. Good blog posts show hurdles that were encountered and explain how they were overcome (not that everything is rainbows and unicorns). 
+- The most interesting posts are ones that teach or explain how to do something in a way maybe others haven’t thought of. Good blog posts show hurdles that were encountered and explain how they were overcome (not that everything is rainbows and unicorns).
 - A few posts that have performed well on the Node.js Foundation Medium page include Walmart Strives to be an Online Retailer with Node.js (use case); Node.js Transitions to LTS (news); The Progress of Node.js Post io.js and Node.js merger (progress update).
-- When showing upstreaming of a patch fixing an issue for others, link back to the Github issue so readers can follow along. 
-- We don’t avoid critical commentary or broad issues, but approach them with sensitivity, professionalism and tact in a way that is beneficial and positive for the community. 
-- Node.js Foundation Medium posts should be no longer than 1,000 words and no shorter than 300 words. 
-
-Diagrams or photos are strongly encouraged. 
+- When showing upstreaming of a patch fixing an issue for others, link back to the Github issue so readers can follow along.
```

Comment:
Is there a case where this line has actually been used? If not, perhaps worth stiriking it entirely?

Additionally, if we don't strike it, it should be updated to `GitHub`

Reply:
I'll fix it as `GitHub`

Reply:
This was resolved but not fixed 😅

Reply:
Done